assets/js/blocks/reviews-by-product/block.js

```diff
@@ -86,6 +147,14 @@ class ReviewsByProduct extends Component {
 						)
 					}
 				</ul>
+				{ totalReviews > reviews.length && (
+					<button
```

Comment:
Should we use a real Button component here?

Reply:
I was wondering the same, my feeling was that considering this is rendered in the frontend, it was better to try to use plain HTML instead of Gutenberg components, but I think it might be worth discussing the pros and cons of each solution.

Looking at `product-categories`, we are using an HTML `<select>` instead of a `<SelectControl />` component, but at the same time we are using `<IconButton />`, so I think there isn't a set-in-stone strategy for this yet.